buildah.go

```diff
@@ -40,75 +38,23 @@ const (
 	stateFile = Package + ".json"
 )
 
-// PullPolicy takes the value PullIfMissing, PullAlways, PullIfNewer, or PullNever.
-type PullPolicy int
+type PullPolicy = define.PullPolicy
 
-const (
-	// PullIfMissing is one of the values that BuilderOptions.PullPolicy
-	// can take, signalling that the source image should be pulled from a
-	// registry if a local copy of it is not already present.
-	PullIfMissing PullPolicy = iota
-	// PullAlways is one of the values that BuilderOptions.PullPolicy can
-	// take, signalling that a fresh, possibly updated, copy of the image
-	// should be pulled from a registry before the build proceeds.
-	PullAlways
-	// PullIfNewer is one of the values that BuilderOptions.PullPolicy
-	// can take, signalling that the source image should only be pulled
-	// from a registry if a local copy is not already present or if a
-	// newer version the image is present on the repository.
-	PullIfNewer
-	// PullNever is one of the values that BuilderOptions.PullPolicy can
-	// take, signalling that the source image should not be pulled from a
-	// registry if a local copy of it is not already present.
-	PullNever
+var (
+	PullIfMissing = define.PullIfMissing
+	PullAlways = define.PullAlways
+	PullIfNewer = define.PullIfNewer
+	PullNever = define.PullNever
 )
 
-// String converts a PullPolicy into a string.
-func (p PullPolicy) String() string {
-	switch p {
-	case PullIfMissing:
-		return "PullIfMissing"
-	case PullAlways:
-		return "PullAlways"
-	case PullIfNewer:
-		return "PullIfNewer"
-	case PullNever:
-		return "PullNever"
-	}
-	return fmt.Sprintf("unrecognized policy %d", p)
-}
-
-// NetworkConfigurationPolicy takes the value NetworkDefault, NetworkDisabled,
-// or NetworkEnabled.
-type NetworkConfigurationPolicy int
+type NetworkConfigurationPolicy = define.NetworkConfigurationPolicy
 
-const (
-	// NetworkDefault is one of the values that BuilderOptions.ConfigureNetwork
-	// can take, signalling that the default behavior should be used.
-	NetworkDefault NetworkConfigurationPolicy = iota
-	// NetworkDisabled is one of the values that BuilderOptions.ConfigureNetwork
-	// can take, signalling that network interfaces should NOT be configured for
-	// newly-created network namespaces.
-	NetworkDisabled
-	// NetworkEnabled is one of the values that BuilderOptions.ConfigureNetwork
-	// can take, signalling that network interfaces should be configured for
-	// newly-created network namespaces.
-	NetworkEnabled
+var (
+	NetworkDefault=define.NetworkDefault
+    NetworkDisabled=define.NetworkDisabled
```

Comment:
gofmt?

Reply:
These should be `const`, like the values they're defined to.